components/BottomNavigation/tests/unit/MDCBottomNavigationBarTests.m

```diff
@@ -560,4 +566,27 @@ - (void)testTraitCollectionDidChangeBlockCalledWhenTraitCollectionChanges {
   XCTAssertEqual(passedTraitCollection, testTraitCollection);
 }
 
+#pragma mark - MDCElevation
+
+- (void)testCurrentElevationMatchesElevationWhenElevationChanges {
+  // When
+  self.bottomNavBar.elevation = MDCShadowElevationQuickEntryResting;
+
+  // Then
+  XCTAssertEqualWithAccuracy(self.bottomNavBar.mdc_currentElevation, self.bottomNavBar.elevation,
+                             0.001);
+}
+
+- (void)testSettingOverrideBaseElevationReturnsSetValue {
+  // Given
+  CGFloat expectedBaseElevation = 99;
+
+  // When
+  self.bottomNavBar.mdc_overrideBaseElevation = expectedBaseElevation;
+
+  // Then
+  XCTAssertEqualWithAccuracy(self.bottomNavBar.mdc_overrideBaseElevation, expectedBaseElevation,
+                             0.001);
+}
+
```

Comment:
please add a test for the elevationDidChangeBlock

Reply:
It's not called by anything - what would you like to test?

Reply:
you can verify that it is correctly stored in the component:
```
- (void)testElevationDidChangeBlockIsCorrectlySetUp {
  // Given
  __block NSNumber *weakElevationDidChangeBlockWasCalled = @NO;
  self.card.mdc_elevationDidChangeBlock = ^(CGFloat elevation) {
    weakElevationDidChangeBlockWasCalled = @YES;
  };

  // When
  self.card.mdc_elevationDidChangeBlock(0);

  // Then
  XCTAssertTrue(weakElevationDidChangeBlockWasCalled.boolValue);
}
```

Reply:
Added functional tests now that the category has been written.